announcements/global_council_election_2020/announcement_general_election.md

```diff
@@ -0,0 +1,17 @@
+# Draft copy for PyLadies General Election announcement
+
+## PyLadies first Global Council Election is now open! (blog post suggestion)
+
+The Python open source community is renowned for being innovative yet friendly and open. We are a part of a large ecosystem of passionate engineers, data scientists, analysts, coordinators and students. If you’re interested in contributing, why not join our first Global PyLadies council.
```

Comment:
Should this say PyLadies community instead of Python open source community?

Reply:
sure